File: examples/code/home/use-tools/call-tools-directly/github_directly.js
import Arcade from "@arcadeai/arcadejs";

const client = new Arcade();

let userId = "{arcade_user_id}";

const response = await client.tools.execute({
  tool_name: "GitHub.SetStarred",
  input: {
    owner: "ArcadeAI",
    name: "arcade-ai",
    starred: true,
  },
  user_id: userId,
});

console.log(response.output.value);

File: examples/code/home/use-tools/call-tools-directly/github_directly.py
from arcadepy import Arcade

client = Arcade()

user_id = "{arcade_user_id}"

response = client.tools.execute(
    tool_name="GitHub.SetStarred",
    input={
        "owner": "ArcadeAI",
        "name": "arcade-ai",
        "starred": True,
    },
    user_id=user_id,
)

print(response.output.value)
